tests/test.py

```diff
@@ -719,11 +721,33 @@ def _connect(self, loop):
                 database=self.connection_args.dbname,
                 **asyncpg_connect_args))
 
+    def _set_text_format(self, conn):
+        """Force text format to numeric types."""
+        loop = asyncio.get_event_loop()
+        for pg_type in ['int2', 'int4', 'int8']:
+            loop.run_until_complete(
+                conn.set_type_codec(pg_type,
+                    schema='pg_catalog',
+                    encoder=str,
+                    decoder=int,
+                    format='text')
+            )
+        for pg_type in ['float4', 'float8']:
+            loop.run_until_complete(
+                conn.set_type_codec(pg_type,
+                    schema='pg_catalog',
+                    encoder=str,
+                    decoder=float,
+                    format='text')
+            )
+
     def execute_prepared_statement(self, query, args=None):
         if not args:
             args = []
         loop = asyncio.get_event_loop()
         conn = self._connect(loop)
+        if self.connection_args.format == 'text':
```

Comment:
can we use some constants instead hardcoded values? it force to assign them somewhere together and we may see which formats are used/supported. otherwise we should search by `.format*` where and which formats used in the code.
And as I understand, text format added here for acra-ee and for now not used in acra-ce, correct?

Reply:
> can we use some constants instead hardcoded values?

Yeah, that's a good idea.

What would be the best way to introduce such constants? I though about adding them to `AsyncpgExecutor` class like this:
```python
class AsyncpgExecutor(QueryExecutor):

    TextFormat = 'text'
    BinaryFormat = 'binary'
```
so that we can do
```python
    def execute_prepared_statement(self, query, args=None):
        conn = self._connect(loop)
        if self.connection_args.format == TextFormat:
            self._set_text_format(conn)
```
and then something like this:
```python
class BaseBinaryPostgreSQLTestCase(BaseTestCase):

    FORMAT = AsyncpgExecutor.BinaryFormat
```

Is this an acceptable approach?

Reply:
>  And as I understand, text format added here for acra-ee and for now not used in acra-ce, correct?

Yes. Acra CE currently doesn't use text format. Well, we can check it in transparent encryption tests, for example. I've been using them for debugging. Obviously, since Acra CE does not modify integers, the tests should behave exactly the same with either format. Should we test this format switch in Acra CE?

Reply:
> Is this an acceptable approach?

Yes, due to missing enum support in python, common way to define class level variables as constants

> Should we test this format switch in Acra CE?

IMHO, will be good. probably it will save us from some mistakes when we will try to touch text format in acra-ce too.

Reply:
I've added both constants and a new test with text format.